docs/core/tools/global-json.md

```diff
@@ -1,32 +1,92 @@
+author: mairaw
 ms.author: mairaw
-ms.date: 04/05/2017
+ms.date: 06/27/2018
+ms.custom: "updateeachrelease"
 ---
 # global.json reference
 
-The *global.json* file allows selection of the .NET Core tools version being used through the `sdk` property.
+[!INCLUDE [topic-appliesto-net-core-all](../../../includes/topic-appliesto-net-core-all.md)]
 
-.NET Core CLI tools look for this file in the current working directory (which isn't necessarily the same as the project directory) or one of its parent directories.
+The *global.json* file allows selection of the .NET Core SDK version being used through the `sdk` property.
+
+.NET Core SDK looks for this file in the current working directory (which isn't necessarily the same as the project directory) or one of its parent directories.
 
 ## sdk
+
 Type: Object
 
-Specifies information about the SDK.
+Specifies information about the .NET Core SDK to select.
 
 ### version
+
 Type: String
 
-The version of the SDK to use.
+The version of the .NET Core SDK to use.
+
+Note that this field:
 
-For example:
+- Doesn't have globbing support, that is, the full version number has to be specified.
+- Doesn't support version ranges.
+
+The following example shows the contents of a *global.json* file:
 
 ```json
 {
   "sdk": {
-    "version": "1.0.0-preview2-003121"
+    "version": "2.1.300"
   }
 }
 ```
+
+## global.json and the .NET Core CLI
+
+Starting with .NET Core SDK 2.1, to verify which SDK versions you've installed to specify in the *global.json* file, you can run the following command:
+
+```console
+dotnet --list-sdks
+```
+
+To install additional .NET Core SDK versions on your machine, visit [All .NET Downloads](https://www.microsoft.com/net/download/all).
+
+You can create a new the *global.json* file in the current directory by executing the [dotnet new](dotnet-new.md) command, similar to the following example:
+
+```console
+dotnet new globaljson --sdk-version 2.1.300
+```
+
+## Matching rules
+
+With .NET Core SDK 1.x, if you specified a version and no exact match was found, the latest installed SDK version was used. Latest SDK version can be either release or pre-release - the highest version number wins.
+
+Starting with .NET Core SDK 2.0, the following rules apply when determining which version of the SDK to use:
+
+- If no *global.json* file is found or *global.json* doesn't specify an SDK version, the latest installed SDK version is used. Latest SDK version can be either release or pre-release - the highest version number wins.
+- If *global.json* does specify an SDK version:
+  - If the specified SDK version is found on the machine, that exact version is used.
+  - If the specified SDK version can't be found on the machine, the latest installed SDK **patch-version** of the specified SDK version is used. Latest installed SDK **patch-version** can be either release or pre-release - the highest version number wins.
```

Comment:
I'm trying to think of a better way to explain this. I think saying "the latest installed SDK patch-version that is greater than the specified version of the specified SDK version is used." is not accurate. 
For example, if I specified 2.1.300 and I have 2.1.401 on my machine, I have an installed SDK patch-version that is greater than the specified one, but that one is not picked because it's not a patch version of the version specified. Makes sense? 🤣

Reply:
Ok, I think this simplifies things: "If the specified SDK version can't be found on the machine, the latest installed SDK **patch-version** of that version is used." Does that look ok to you?